src/lang/en-US.json

```diff
@@ -118,6 +118,11 @@
   "barracks_value_2048": "Radiant Top Ranged",
 
   "benchmarks_description": "%d %s is equal or higher than %d% of recent performances on this hero",
+  "benchmarks_xp_per_min": "XP per Minute",
```

Comment:
i think the rest of these strings also have existing versions that you could reuse as well.  Didn't notice it before I merged, if you want to you can submit a PR to do that @bdbch